nixos/modules/services/networking/opkssh/opkssh.nix

```diff
@@ -0,0 +1,171 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+with lib;
+
+let
+  cfg = config.services.opkssh;
+  opkssh = pkgs.opkssh;
```

Comment:
`package` is (correctly) defined as an option below so this variable doesn't seem to be used